lib/kwtool_web/router.ex

```diff
@@ -23,6 +23,19 @@ defmodule KwtoolWeb.Router do
     plug :accepts, ["json"]
   end
 
+  scope "/api/v1/", KwtoolWeb.Api.V1 do
+    pipe_through [
+      :api,
+      KwtoolWeb.CheckEmptyBodyParamsPlug
+    ]
+  end
+
+  scope "/api/v1", KwtoolWeb.Api.V1, as: :api do
+    pipe_through :api
+
+    post "/sign_in", SessionController, :create
+  end
```

Comment:
Does this work? 🤔 💡 

```suggestion
  scope "/api/v1", KwtoolWeb.Api.V1, as: :api do
    pipe_through [
      :api,
      KwtoolWeb.CheckEmptyBodyParamsPlug
    ]

    post "/sign_in", SessionController, :create
  end
```

Reply:
Updated in https://github.com/byhbt/kwtool/pull/45/commits/5c56d403ac73dd8c41a089f0f6397895627d3386